content_scripts/link_hints.coffee

```diff
@@ -289,7 +292,7 @@ class LinkHintsMode
     else
       @tabCount = previousTabCount if event.ctrlKey or event.metaKey or event.altKey
       unless event.repeat
-        if keyChar = KeyboardUtils.getKeyChar(event).toLowerCase()
+        if keyChar = KeyboardUtils.getKeyChar(event)
```

Comment:
We still want `toLowerCase` applied when `Settings.get "filterLinkHints"` is falsey, otherwise it breaks <kbd>shift</kbd> for 'alphabet hints'.